repos/system_upgrade/el7toel8/actors/checkchrony/actor.py

```diff
@@ -0,0 +1,53 @@
+from leapp.actors import Actor
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.libraries.common.reporting import report_generic
+from leapp.libraries.stdlib import CalledProcessError, run
+
+
+class CheckChrony(Actor):
+    """
+    Check for incompatible changes in chrony configuration.
+
+    Warn that the default chrony configuration in RHEL8 uses the leapsectz
+    directive.
+    """
+
+    name = 'check_chrony'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        installed_packages = set()
+
+        signed_rpms = self.consume(InstalledRedHatSignedRPM)
```

Comment:
Isn't common practice with the actors to use next(self.consumes) instead of directly referencing class?

Reply:
@fernflower you are confusing something
next(self.consume(...)) maybe is used mostly where they want at most one message.

Here the usage is perfectly valid. They get an iterable (at the moment this is a generator) with all messages that are of type InstalledRedHatSignedRPM

Then they iterate over them below (see for loop)